src/Altinn.Broker.API/Controllers/FileTransferController.cs

```diff
@@ -78,7 +82,10 @@ CancellationToken cancellationToken
             ContentLength = Request.ContentLength ?? Request.Body.Length
         }, cancellationToken);
         return commandResult.Match(
-            fileId => Ok(fileId.ToString()),
+            fileId => Ok(new FileTransferUploadResponseExt()
+            {
+                FileId = fileId
+            }),
```

Comment:
Consider correcting this naming.
"fileId" is an outdated term, the correct name is "fileTransferId"
(In this case the property name was not previousy exposed in the API, but with this change it becomes; so therefore naming becomes importan).